documentation/book/con-custom-resources-status.adoc

```diff
@@ -6,15 +6,64 @@
 
 = {ProductName} custom resource status
 
-The `status` property of a {ProductName}-specific custom resource publishes the _current_ state of the resource to users and tools that need the information.
+The `status` property of a {ProductName} custom resource publishes information about the resource to users and tools that need it.
 
-Status information is useful for tracking progress related to a resource achieving its _desired_ state, as defined by the `spec` property. The `status` provides the time and reason the state of the resource changed and details of events preventing or delaying the Operator from realizing the desired state.
+Several resources have a `status` property, as described in the following table.
 
-{ProductName} creates and maintains the status of custom resources, periodically evaluating the current state of the custom resource and updating its status accordingly.
+[cols="3*",options="header",stripes="none",separator=¦]
+|===
 
-When performing an update on a custom resource using `kubectl edit`, for example, its `status` is not editable. Moreover, changing the `status` would not affect the configuration of the Kafka cluster.
+m¦{ProductName} resource
+¦Schema reference
+¦Publishes status information on...
+
+m¦Kafka
+¦xref:type-KafkaStatus-reference[]
+¦The Kafka cluster.
+
+m¦KafkaConnect
+¦xref:type-KafkaConnectStatus-reference[]
+¦The Kafka Connect cluster, if deployed.
+
+m¦KafkaConnectS2I
+¦xref:type-KafkaConnectS2Istatus-reference[]
+¦The Kafka Connect cluster with Source-to-Image support, if deployed.
+
+m¦KafkaMirrorMaker
+¦xref:type-KafkaMirrorMakerStatus-reference[]
+¦The Kafka MirrorMaker tool, if deployed.
+
+m¦KafkaUser
+¦xref:type-KafkaUserStatus-reference[]
+¦Kafka users in your cluster.
+
+m¦KafkaBridge
+¦xref:type-KafkaBridgeStatus-reference[]
+¦The {ProductName} Kafka Bridge, if deployed.
+
+|===
+//When dev work is complete, add `KafkaTopicStatus` to the above table. `KafkaTopicStatus`: Kafka topics and their partitions
```

Comment:
should you remove this comment from the doc?

Reply:
Yes, I'll remove it.